src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -38,6 +38,7 @@
     <PackageReference Include="PSDesiredStateConfiguration" Version="6.0.0-beta.8" />
     <PackageReference Include="PowerShellHelpFiles" Version="1.0.0-*" />
     <PackageReference Include="psrp.windows" Version="6.1.0-*" />
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="2.0.0" />
```

Comment:
I think this reference should be put in `SDK.csproj` with a `Condition` attribute, like `<PackageReference Include="Microsoft.Windows.Compatibility" Version="2.0.0" Condition=" '$(IsWindows)' != 'true' " />`. In this way, application that refernce to `SDK` nuget package will get those extended pack too.

`IsWindows` is defined in https://github.com/PowerShell/PowerShell/blob/master/PowerShell.Common.props#L113

Reply:
/cc @adityapatwardhan, in case I'm wrong about the SDK reference part.

Reply:
Actually, I don't know if we should even have the `condition` part. Some of the compat assemblies work on Unix platforms too.

Reply:
Yes, this should be part of `SDK`, so apps hosting PowerShell can use the modules with WCP.

Reply:
Perhaps we don't need the condition as @daxian-dbw is correct that some of the assemblies from the WCP meta-package work on non-Windows so we should make those available.  I'll make the change.